### PR TITLE
Guess Who Tobias Hagberg, Jan 21

### PR DESCRIPTION
**Pull request by TobiasHagberg**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -254,90 +255,119 @@ const start = () => {
...
   // remember the confirm() ?
   // If the player wants to guess, invoke the checkMyGuess function.
 }
 
 // If you confirm, this function is invoked
 const checkMyGuess = (suspect) => {
+  if (suspect === secret.name) {
+    alert('That is correct, you win!')
+    board.innerHTML = ''
+    winOrLose.style.display = 'block'
```
Comment (sirimatsdotter) [inline code/script.js:356]: So this triggers an alert, but I cannot see the winorloose section. I think perhaps that you should exclude the alert and instead place your message inside the win or loose section by using .innerHTML.

File: code/script.js [hunk 2]
```diff
@@ -254,90 +255,119 @@ const start = () => {
...
 }
-
 // Invokes the start function when website is loaded
 start()
 
 // All the event listeners
 restartButton.addEventListener('click', start)
+questions.addEventListener('change', selectQuestion)
+findOutButton.addEventListener('click', checkQuestion)
+playAgainButton.addEventListener('click', start)
```
Comment (sirimatsdotter) [inline code/script.js:373]: The play again button is not visible since the win or loose section never appears

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #BDABBE;
+  --secondary: #32575D;
```
Comment (sirimatsdotter) [inline code/style.css:4]: I loved learning about how one can set key colors using --primary and --secondary - makes it so much easier to change the look in just a few seconds!

File: README.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+In this week's project you'll create your own digital version of the old character guessing board game Guess Who?
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The approach was slow but steady, tried to take in as much as possible of provided material and made a handdrawn sketch of the flow before attempting it in code. I follewed the step by step guide and made my way through the solutions/problems. The game did'n feel complete without the play again functionalty so added that as one of the bonus steps.
```
Comment (sirimatsdotter) [inline README.md:7]: I think it's a great idea to sketch it first by hand 😍

File: README.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+
+I find myself always wanting an perhaps needing more time. all and all, I'm happy that my game is working. If I have had more time I would have put it into styling and extra effects for more fun
```
Comment (sirimatsdotter) [inline README.md:9]: I think we all need more time 😅 But you have made the game work which is a great achievement by itself!

File: README.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 
 ## View it live
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://gracious-ritchie-308aae.netlify.app/
```
Comment (sirimatsdotter) [inline README.md:13]: It's looking great, but I can't see the win or loose section, instead there is an alert message

**Review by sirimatsdotter — CHANGES_REQUESTED** [review]
Great job Tobias, the game works great and I like the color you chose!🙂

Not sure if this was a mandatory requirement but  the win or loose section does not display after doing the final guess (I will leave that for the code coaches to judge if this was a mandatory step), but I think it could be a good idea to try to implement if you have time. It seems like you tried to display the section, but at the same time you used an alert which is a popup and won't display the message in the win or loose section...

Also, after the final guess the page does not reload, which might be a good idea to implement for the user. I saw that you wrote about that you implemented the "play again" in your readme, but I cannot see that it is implemented on your live site.

File: code/script.js [hunk 7]
```diff
@@ -254,90 +255,119 @@ const start = () => {
...
   // store the interaction from the player in a variable.
   // remember the confirm() ?
   // If the player wants to guess, invoke the checkMyGuess function.
 }
 
 // If you confirm, this function is invoked
 const checkMyGuess = (suspect) => {
+  if (suspect === secret.name) {
+    alert('That is correct, you win!')
+    board.innerHTML = ''
```
Comment (KarolCislowski) [inline code/script.js:355]: `bord.style.display='none'` would do the magic 🧙🏻

File: code/script.js [hunk 8]
```diff
@@ -254,90 +255,119 @@ const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  winOrLose.style.display = ''
```
Comment (KarolCislowski) [inline code/script.js:260]: If you decide to make changes I suggested at the end of the code, you need to make some here as well:
```
winOrLose.style.display = 'none'
board.style.display='flex'
```

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 Hope my tips will help you with your minor issues 🤓 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**